### PR TITLE
Zero data component

### PR DESCRIPTION
**Pull request by rakol-ms**
Signed-off-by: Rajendra Kolli <rakol@microsoft.com>

no_info
no_svc
no_workloads
svc_no_pods

### CODE REVIEW

**Comment by madhuv-msft** [comment]
when will the following UI be shown?

https://user-images.githubusercontent.com/45386516/51603394-b27d7080-1f2f-11e9-9f3c-00ce0532a068.jpg

**Comment by madhuv-msft** [comment]
keep PNGs in one folder, rather keeping them in webui folder itslef, foldername could be, webui\static etc or src\static etc

**Comment by rakol-ms** [comment]
> when will the following UI be shown?
> 
> https://user-images.githubusercontent.com/45386516/51603394-b27d7080-1f2f-11e9-9f3c-00ce0532a068.jpg

@madhuv-msft This screen shown when there are no workloads *and* no services. Something on the lines of an empty cluster

File: src/WebUI/Components/ZeroDataComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+export interface IZeroDataComponentProps extends IVssComponentProperties {
+    title?: string,
+    hyperLink?: string,
+    textline1?: string,
+    hyperLinkLabel?: string,
+    textline2?: string
+    imagePath?: string,
+    imageAltText?: string
+    className?: string
```
[the remark this answers is not in the document]
Comment (rakol-ms) [reply src/WebUI/Components/ZeroDataComponent.tsx]: This is used as card title.

File: src/WebUI/Components/ZeroDataComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                        imageAltText={this.props.imageAltText || ""}
+                        imagePath={this.props.imagePath || ""}
+                        secondaryText={this.__getTextArea()}
+                    />
+                </Card>
+            </div>
+        );
+    }
+
+    private __getTextArea(): JSX.Element {
```
[the remark this answers is not in the document]
Comment (rakol-ms) [reply src/WebUI/Components/ZeroDataComponent.tsx]: done.

File: src/WebUI/Components/ZeroDataComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import { IVssComponentProperties } from '../Types';
+import { BaseComponent, css } from "@uifabric/utilities/lib";
+import { Link } from 'azure-devops-ui/Link';
+import './Common.scss';
+import './ListComponent.scss'
+
+export interface IZeroDataComponentProps extends IVssComponentProperties {
+    title?: string,
+    hyperLink?: string,
+    textline1?: string,
```
[the remark this answers is not in the document]
Comment (rakol-ms) [reply src/WebUI/Components/ZeroDataComponent.tsx]: title is used as Card title. Please refer to service details image where there is a title "Associated Pods"

**Pull request merged**